scipy/odr/__init__.py

```diff
@@ -128,6 +124,7 @@ def f(B, x):
 
 from .odrpack import *
 from .models import *
+from . import add_newdocs
```

Comment:
This will show up in the `odr` namespace, which is undesired. Do you need the import? If so, can you either `del` it after use or import it `as _add_newdocs`? But I think it's not needed, no other module `__init__.py` files have it. Normal structure is:

```
.. currentmodule:: scipy.odr

.. autosummary::
   :toctree: generated/

   <function and object name listing>
```

Reply:
Thank you for your help. 
The `odr.odr()` function page of current master branch is empty: http://scipy.github.io/devdocs/generated/scipy.odr.odr.html#scipy.odr.odr
But, the `add_newdocs.py` has the docstring of it. When I import it in the `__init__.py`, it seems that the doc import the docstring correctly:
https://19023-1460385-gh.circle-artifacts.com/0/html-scipyorg/generated/scipy.odr.odr.html#scipy.odr.odr
Do you have any other idea to import the docstring correctly?

Reply:
Ah I see, yes that looks good. Then keeping the import but using an underscore or `del _add_newdocs` right before populating `__all__` would be good.

Reply:
OK. I changed the import using an underscore. This PR is almost ready for review, but I need time, because I am not sure what is a good example for odr.odr() function. I could not find any code in GitHub public repos using this function...

Reply:
I think it's fine to not add an example for `odr`, it's a low-level function, users should use the `ODR`, `Model`, etc. classes. This PR looks ready to me.